phasta/phPartition.cc

```diff
@@ -139,9 +155,9 @@ void parmaTet(Input& in, apf::Mesh2* m, bool runGap) {
 }
 
 void parmaBalance(Input& in, apf::Mesh2* m, bool runGap) {
-  if( isMixed(m) )
-    parmaMixed(in,m);
-  else
+//  if( isMixed(m) )
+//    parmaMixed(in,m);
+//  else
```

Comment:
I assume this should be removed?

Reply:
Add a flag to disable the call to `parmaMixed(...)` and probably a print statement.